provision/install/python.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+set -x
+
+apt-get install -y python2.7
+
+# Install the latest version of pip
+apt-get install -y curl ca-certificates
+curl -SL 'https://bootstrap.pypa.io/get-pip.py' | python
```

Comment:
It does... I'm also trying to avoid installing unnecessary stuff as the `python-pip` package has quite a few dependencies. But I think I can find a better way...